examples/python/controllers/pidTwoZones.py

```diff
@@ -74,11 +79,41 @@ def initialize():
         Defines the control input to be used for the next step.
         {<input_name> : <input_value>}
 
-    '''
+    """
 
-    u = {'oveActNor_u':0,
-         'oveActNor_activate': 1,
-         'oveActSou_u':0,
-         'oveActSou_activate': 1}
+    u = {
+        'oveActNor_u': 0,
+        'oveActNor_activate': 1,
+        'oveActSou_u': 0,
+        'oveActSou_activate': 1
+    }
 
     return u
+
+
+def update_predictions(prediction_config, forecast):
```

Comment:
I suggest using "forecast" terminology instead of "prediction" terminology to maintain consistency with the BOPTEST forecast api and so it is easier for a reader to associate the connection with it.

Reply:
As suggested, I changed it into "update_forecast"

Reply:
My suggestion was to change all references to "prediction" to "forecast," including in the controller modules (e.g. predictions_store, testcase1.prediction_store, Controller.prediction, etc.)